### PR TITLE
[5478] AaAdmin I can take actions on a granted application coming from partners without a bankAccount id for the payout

### PR DESCRIPTION
**Pull request by JacobNyman**

### CODE REVIEW

File: partners.yml [hunk 1]
```diff
@@ -457,6 +457,10 @@ components:
           type: boolean
           description: Indicate if customer accepted can be profiled by Moank to take a credit decision
           example: true
+        bankAccountId:
+          type: string
+          description: Tink code fetched with the Tink link.
+          example: '123456789'
```
Comment (DavidVergnaultMoank) [inline partners.yml:463]: The bank account id is suppoed to be a mongoId of one of the bank accounts of the user 🤔 Why did you think it was that ? 😅

**Review by DavidVergnaultMoank — CHANGES_REQUESTED** [review]

File: partners.yml [hunk 2]
```diff
@@ -457,6 +457,10 @@ components:
           type: boolean
           description: Indicate if customer accepted can be profiled by Moank to take a credit decision
           example: true
+        bankAccountId:
+          type: string
+          description: Tink code fetched with the Tink link.
```
Comment (FabriSilve) [inline partners.yml:462]: Here it should not be the tink code fetched previously but it should be the id of the account chosen by the customer from the list of the account we are returning when they call the `send-code` endpoint ;)

**Review by FabriSilve — CHANGES_REQUESTED** [review]

File: partners.yml [hunk 3]
```diff
@@ -457,6 +457,10 @@ components:
           type: boolean
           description: Indicate if customer accepted can be profiled by Moank to take a credit decision
           example: true
+        bankAccountId:
+          type: string
+          description: Unique identifier of the bank account received from the Send Tink Code endpoint.
+          example: '12345'
```
Comment (DavidVergnaultMoank) [inline partners.yml:463]: of "one of the bank accounts" I would say

File: partners.yml [hunk 4]
```diff
@@ -457,6 +457,10 @@ components:
           type: boolean
           description: Indicate if customer accepted can be profiled by Moank to take a credit decision
           example: true
+        bankAccountId:
+          type: string
+          description: Unique identifier of the bank account received from the Send Tink Code endpoint.
```
Comment (FabriSilve) [inline partners.yml:462]: I would like to rephrase it a little bit, so the understand what is the usage of it 🤔  Does something like the following makes sense for you? 🤔 
`Unique identifier of the bank account for the payout chosen by the customer from the list of accounts returned by the "/bank/send-code" endpoint`

Comment (JacobNyman) [reply partners.yml:462]: Yes that seems to be more descriptive!

**Review by FabriSilve — APPROVED** [review]

**Pull request merged**